blocks/components/editable/index.js

```diff
@@ -349,6 +350,8 @@ export default class Editable extends wp.element.Component {
 		this.setState( {
 			formats: merge( {}, this.state.formats, formats )
 		} );
+
+		this.editor.setDirty( true );
```

Comment:
I had to re-add setDirty as it was not saving on Chrome after merging the latest changes from master without it.
Note my test procedure was:
1. select a word in unordered list
2. click bold button
3. click button to change to ordered list